scripts/mods/mod_reforged/dynamic_spawns/parties/barbarian_parties.nut

```diff
@@ -1,3 +1,5 @@
+// still need to figure out frost unhold spawns and adjust spawns after that resource value
```

Comment:
this should be removed as the biggest issue with the frost unholds in barbarian parties has been fixed and merged